runexample

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
It would be valuable to have some convenience scripts. I can add documentation to examples/README.me and experiments/pipeline/README.md, possibly the wiki as well.